### PR TITLE
added upvote and downvote for blogs

### PR DESCRIPTION
**Pull request by nik72619c**
@Swap76 please have a look!

### CODE REVIEW

File: src/controllers/blogs/BlogController.js [hunk 1]
```diff
@@ -197,3 +197,55 @@ module.exports.checkBlogOwner = async (req, res, next) => {
...
 };
+
+/**
+ * method for upvoting a blog
+ * @route /blog/:id/upvote
+ * @param id blog post id
+ * @method POST
+ */
+module.exports.upvote = async (req, res) => {
+  const id = req.params.id;
```
Comment (Swap76) [inline src/controllers/blogs/BlogController.js]: Good clean and indented code

File: src/controllers/blogs/BlogController.js [hunk 2]
```diff
@@ -197,3 +197,55 @@ module.exports.checkBlogOwner = async (req, res, next) => {
...
+ * @method POST
+ */
+module.exports.downvote = async (req, res) => {
+  const id = req.params.id;
+  if (!validator.isMongoId(id)) {
+    res.status(400).send({'error':'There is no such blog post'});
+  } else{
+
+    try {
+      const result = await Blog.findByIdAndUpdate(id, { $inc: {downvote : 1 } });
```
Comment (Swap76) [inline src/controllers/blogs/BlogController.js]: Hey @nik72619c 

I need the userID into the upvotes as a array of userID

File: src/models/Blogs.js [hunk 3]
```diff
@@ -14,6 +14,14 @@ var blogSchema = new Schema ({
     text: true,
     maxlength: 256,
   },
+  upvote: {
+    type: Number,
```
Comment (Swap76) [inline src/models/Blogs.js]: Here want the array of userId of users

**Comment by nik72619c** [comment]
okay, will apply the changes !

**Comment by Swap76** [comment]
@nik72619c The work you have done for the project is highly appreciated. Thanks :-)

Looking forward to working with you on my new project related to Deno https://github.com/Swap76/Learn-Deno/ and together making this better.

Star to the repo is highly appreciated.

**Pull request closed without merging**